client/my-sites/stats/hooks/use-subscribers-overview.ts

```diff
@@ -20,6 +20,15 @@ function getLabels( dateToQuery: number ) {
 	}
 }
 
+function getNote( heading: string ) {
+	let note = translate( 'As of today' );
+	if ( heading !== 'Today' ) {
```

Comment:
I think this comparison will fail in a non-English environment. The value as returned by `getLabels()` is translated so we probably want to compare with the translated string.

Maybe we could set it up as a const outside the functions?

```
const translatedToday = translate( 'Today' );
```

Otherwise, I like this refactoring. Makes sense to have processing of the labels & notes co-located like this.

Reply:
That's a nice catch. Thanks for the suggestion. I've added a constant to be sure we are comparing the translated strings.